views/templates/front/mailalerts-account-line.tpl

```diff
@@ -23,15 +23,17 @@
 * International Registered Trademark & Property of PrestaShop SA
 *}
 
+<a href="#"
+   title="{l s='Remove mail alert' d='Modules.Mailalerts.Shop'}"
+   class="js-remove-email-alert btn btn-link"
+   rel="js-id-emailalerts-{$mailAlert.id_product|intval}-{$mailAlert.id_product_attribute|intval}"
+   data-url="{url entity='module' name='ps_emailalerts' controller='actions' params=['process' => 'remove']}"
+   style="float:right">
+  <i class="material-icons">delete</i>
+</a>
 <a href="{$mailAlert.link}">
-  <img src="{$mailAlert.cover_url}" alt=""/>
-  {$mailAlert.name}
-  <span>{$mailAlert.attributes_small}</span>
-  <a href="#"
-     title="{l s='Remove mail alert' d='Modules.Mailalerts.Shop'}"
-     class="js-remove-email-alert btn btn-link"
-     rel="js-id-emailalerts-{$mailAlert.id_product|intval}-{$mailAlert.id_product_attribute|intval}"
-     data-url="{url entity='module' name='ps_emailalerts' controller='actions' params=['process' => 'remove']}">
-    <i class="material-icons">delete</i>
-  </a>
+  <img src="{$mailAlert.cover_url}" alt="" class="mr-1" style="float:left; height: 98px; width: 98px;"/>
```

Comment:
Same here for `float-left`. `98px` is the default size of small thumbnails. Any suggestion on how to make it better than hard code the size?

Reply:
As this float is a BC anyway, if you want to do it properly, refactor it using flexbox and keep the img size natural?

Reply:
I refactored it. Have a look.